front-end/src/components/Skulpt.jsx

```diff
@@ -56,6 +60,23 @@ class Skulpt extends React.Component {
     render() {
         return (
             <div className="ide-card">
+<<<<<<< HEAD
+            <Card >
+                <CardHeader>Python Text Editor</CardHeader>
+                <CardBody>
+                <div>
+                <InputGroup size="lg" className="mock-input">
+                <MyCodeMirror></MyCodeMirror>
+                </InputGroup>
+                <textarea readonly="true" class="output" id="code-output"></textarea>
+                </div>
+                </CardBody>
+            </Card>
+            <div className="submit-btns">
+            <Button color="info" size="md" onClick={this.execute}>Run</Button>{' '}
+            <Button color="outline-info" size="md">Submit</Button>{' '}
+            </div>
+=======

```

Comment:
```suggestion
              //  <MyCodeMirror></MyCodeMirror>
              //  <textarea readonly="true" class="output" id="code-output"></textarea>
```
This is the only new part so it should be added, but I suggest you comment it out and then another commit is done to main that merges this with the new ant components below